test/rulesets/LinearAlgebra/symmetric.jl

```diff
@@ -18,22 +18,33 @@
         @testset "rrule" begin
             # on old versions of julia this combination doesn't infer but we don't care as
             # it infers fine on modern versions.
-            check_inferred = !(VERSION <= v"1.5" && T <: ComplexF64 && SymHerm <: Hermitian)
+            check_inferred = true#!(VERSION <= v"1.5" && T <: ComplexF64 && SymHerm <: Hermitian)
```

Comment:
delete this line and the comment above?

Reply:
I first wanted to check, whether we still needed to exempt older Julia versions here. When CI finishes, I am going to change this accordingly.